src/createStreamingManager.ts

```diff
@@ -114,13 +114,27 @@ export async function createStreamingManager<T extends CreateStreamOptions>(
 
             if (event === StreamEvents.StreamStarted) {
                 console.log('StreamStarted', event, data);
+                analytics?.track('agent-video', {
+                    event,
+                    rtcStats: data ?? [] 
```

Comment:
rtcStats should be removed from "agent-video":start
its only available in "agent-video":stop